unix/lib/time.mli

```diff
@@ -18,6 +18,6 @@ type +'a io = 'a Lwt.t
 
 (** Timeout operations. *)
 
-val sleep : float -> unit Lwt.t
```

Comment:
- It's used in lots of places and there's no need to break everything.
- OCaml uses floats to represent times, so having something that accepts OCaml times is useful.
- Using floats allows representing very long or short durations without errors.
- Using floats avoids error-prone scaling (e.g. all the `* 1_000_000_000` magic constants in the recent patches).
- Using floats avoids overflow errors when converting between timebases (e.g. `duration_at_freq_1 /. freq_1 *. freq_2` works; using `int64` might not, depending on the frequencies and duration).

In general, it's conveniet, familiar and safe. So for code that doesn't care about performance, I'd prefer to stick to floats.